packages/ember-simple-auth/tests/unit/session-stores/shared/cookie-store-behavior.js

```diff
@@ -311,7 +302,8 @@ export default function(options) {
       );
     });
 
-    it('clears cached expiration times when setting expiration to null', function() {
+    // eslint-disable-next-line require-await
+    it('clears cached expiration times when setting expiration to null', async function() {
```

Comment:
Is this `async` in fact necessary?

Reply:
Yeah, otherwise the mocha test is failing, it's throwing some error saying `done() wasn't called after the test finished` and it's failing because of that. The test works correctly, the expectations inside in fact are working, though for some reason I wasn't able to debunk the test has to be async.

Reply:
This sounds like we're working around something – this test itself has nothing asynchronous. We should look into this a bit closer – happy to pair on it next week.

Reply:
OK 👍 
I was able to remove it from one of the places, I figured that it was caused by beforeEach calling an async method on store.
Though can't find the cause for the other one so far, although I wouldn't be surprised if it actually be caused by mocha `setupTest` which expects the hook callback to be `thenable` https://github.com/emberjs/ember-mocha/blob/master/addon-test-support/ember-mocha/setup-test.js#L21

Reply:
Ok, let's figure out how we can fix this one as well.

Reply:
Yeah, for now I've removed the eslint disable line, so it fails PR until it's fixed.